### DISCUSSION TITLE
Show Set Ampere via status endpoint

### DISCUSSION THREAD

**Discussion by pkuehnel** [tobiasehlert/teslamateapi#140] [Q&A]
Is there a way to get the info of the set Ampere within the status endpoint. Currently under Charger there is only charger_actual_current visible but not the configured current in the app neither (as far as I know) the max amp which can be delivered by the Wallbox. Is this an 'issue' related to TeslaMate or related to TeslaMate Api?

**Comment by tobiasehlert** [comment]
hi @pkuehnel,

The set ampere is something that is not available in MQTT ([list of current MQTT topics](https://docs.teslamate.org/docs/integrations/mqtt/)).
This would need to be added to TeslaMate first, before we can add this to the TeslaMateApi status endpoint 😃 

So head over to [TeslaMate](https://github.com/adriankumpf/teslamate) and make a suggestion or maybe even create a PR that adds the functionality.

Regards,
Tobias

**Reply by pkuehnel** [reply]
Ok, thank you I created a [discussion](https://github.com/adriankumpf/teslamate/discussions/2412).

**Comment by pkuehnel** [comment]
@tobiasehlert There should be two new topics in TeslaMate (see linked discussion). Could you please add these two topics to your API?

**Reply by tobiasehlert** [reply]
Absolutely :)

**Reply by tobiasehlert** [reply]
@pkuehnel https://github.com/tobiasehlert/teslamateapi/pull/148 :)
